ui/src/containers/VolumeInformation.js

```diff
@@ -127,13 +155,14 @@ const VolumeInformation = props => {
         <InformationSpan>
           <InformationLabel>{intl.messages.path}</InformationLabel>
           <InformationValue>
-            {pV && pV.spec && pV.spec.local && pV.spec.local.path}
+            {volume?.spec?.rawBlockDevice?.devicePath ||
+              intl.messages.not_applicable}
```

Comment:
The logic here works but is not very future proof.
But since I don't see how it could fail for now, let's keep it (will probably need to be reworked when we have more than two types though).